tests/PriorityMapTest.php

```diff
@@ -89,20 +91,37 @@ public function testGetKeys()
         $list->set('k4', 'v4', 1);
         $list->set('k5', 'v5', 4);
 
-        $this->assertEquals(
+        $result = $list->getKeys();
+        $resultValues = array_values($result);
```

Comment:
where do you use this variable?

Reply:
This variable is unused and I will remove it.